<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PalChromaDecoderConfigDialog</class>
 <widget class="QDialog" name="PalChromaDecoderConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PAL Chroma-Decoder Configuration</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="blackAndWhiteCheckBox">
       <property name="text">
        <string>Black and white</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="twoDeeTransformCheckBox">
       <property name="text">
        <string>Use Transform PAL 2D filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="thresholdModeCheckBox">
       <property name="text">
        <string>Use threshold comparison mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Transform threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="thresholdHorizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="thresholdValueLabel">
       <property name="text">
        <string>0.4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showFFTsCheckBox">
       <property name="text">
        <string>Overlay FFT visualisation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="simplePALCheckBox">
       <property name="text">
        <string>Use Simple PAL decoder</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
